plugins/eventMacro/eventMacro/Runner.pm

```diff
@@ -1814,6 +1776,10 @@ sub resolve_statement {
 	$first =~ s/^"(.+)"$/\1/;
 	$last  =~ s/^"(.+)"$/\1/;
 	
+	#remove backslashes
```

Comment:
on blocks (if , elsif, switch,case) if you use a escaped quote, the '\' would still be there at the end. This fix that